amical/mf_pipeline/bispect.py

```diff
@@ -1037,6 +1054,7 @@ def extract_bs(
     unbias_v2=True,
     compute_cp_cov=True,
     expert_plot=False,
+    save=False,
```

Comment:
this looks like an early merge from a different PR ?
In any case, new arguments should be appended at the end of the signature.
Also, new arguments, especially boolean, should be made keyword only